groovy-language-server/src/main/java/com/palantir/groovylanguageserver/GroovycWrapper.java

```diff
@@ -64,9 +72,15 @@
     private static final Logger log = LoggerFactory.getLogger(GroovycWrapper.class);
 
     private static final String GROOVY_EXTENSION = "groovy";
+    private static final String GROOVY_DEFAULT_INTERFACE = "groovy.lang.GroovyObject";
+    private static final String JAVA_DEFAULT_OBJECT = "java.lang.Object";
+
     private final Path workspaceRoot;
     private final CompilationUnit unit;
+    // Maps from source file -> set of symbols in that file
     private Map<String, Set<SymbolInformation>> fileSymbols = Maps.newHashMap();
+    // Maps from type -> set of references of this type
+    private Map<String, Set<SymbolInformation>> references = Maps.newHashMap();
```

Comment:
`classReferences` wouldn't really reflect that is contained since it also includes interfaces and enums. Anything that is a type.